docs/pages/deploy-a-cluster/deployments/aws-active-passive-deployment.mdx

```diff
@@ -0,0 +1,111 @@
+different cloud regions. These components utilize a global DynamoDB storage backend for the cluster state and audit 
+logs, while a copied S3 bucket is used for session recordings. The Route53 DNS failover is the control point to switch 
+between the active and passive clusters during the event of a regional outage. This document mainly focuses on Kubernetes,
+but same concepts applies with Virtual Machines/EC2 instances.
+
+## Overview
+
+![Diagram showing this Teleport
+architecture](../../../img/deploy-a-cluster/aws-multi-region-active-passive-ha-deployment.png)
+
+### Key Components
+- Two sets of Teleport Auth and Proxy components are deployed in separate regions. In this example, the us-west-1 (Primary)
+  region Teleport cluster is the Active cluster, and the us-east-1 (Secondary) region Teleport cluster is the Passive cluster.
+- All teleport cluster components are deployed as autoscaling groups in both regions.
+- DynamoDB is used for cluster state and audit log storage.
+  - Enable the us-west-1 (Active Region) DynamoDB table with global replication in the us-east-1 (Passive Region) to ensure cluster 
+    state and audit logs are maintained in both regions.
+  - DynamoDB [Global Replication Steps](https://aws.amazon.com/blogs/aws/new-convert-your-single-region-amazon-dynamodb-tables-to-global-tables/)
+-  S3 buckets are for session recording storage.
+  - Enable cross-region replication on the S3 bucket to ensure objects are replicated between both the Active and 
+    Passive regions.
+  - S3 [Replication Steps](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html).
+  - Make sure proper IAM permissions are in place for the successful [Replication of 
+  S3 buckets](https://docs.aws.amazon.com/AmazonS3/latest/userguide/setting-repl-config-perm-overview.html).
+- Configure the Teleport cluster Route53 records to use the 
+  [Failover Routing Policy](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-failover-types.html#dns-failover-types-active-passive) 
+  with the active cluster loadbalancer as the primary record and the passive cluster loadbalancer as the secondary record.
+  - Use health checks to determine Teleport cluster availability. Route53 will automatically send the traffic to the 
+    passive cluster during the event of a regional outage and it will automatically routes the traffic back to the active 
+    cluster once the Active cluster when health checks report a normal state.
+
+### Advantages of this deployment architecture
+
+- Having a cluster in multi-regions makes the clusters quickly available during the event of regional outages.
+- Downtime can be minimized vs traditional disaster recovery from a backup.
+- All required Teleport components can be provisioned within the AWS ecosystem.
+- High-availability Auto Scaling group of Auth Service pods that must remain in a Primary region
+- High-availability Auto Scaling group of Proxy Service pods deployed across multiple regions
```

Comment:
Again, ASGs aren't really relevant here if we're talking about Kubernetes.

Reply:
Okie. I will remove those two points.